content/rule_book/roles_and_responsibilities.md

```diff
@@ -2,68 +2,96 @@
-    - On a Q/Q2V, CVR/MACVR, CR/MACR question, if either the chapter or verse number were not completely finished when the quizzer jumped, the quizmaster will say "More".
-    - On a Reference type question, if the quizzer gives a reference question that is different than the written question, the quizmaster and answer judge must ensure that the reference question given is the exact same type as the written question.
-    - Consult with the answer judge(s) at any time they feel is necessary, provided it does not infringe upon the 30 seconds allotted the quizzer.
-    - If an explanation of a ruling is requested by a captain, the quizmaster will ask if any team wants to challenge before the explanation is given.
+- Rule immediately upon an obviously correct or incorrect answer if there is no indication from the answer judge to the contrary
+
+When a quizzer's response contains a required completion of the question and that completion is different than the written question, the quizmaster will discuss it with the answer judge. If the quizzer gives a reference question that is different than the written question, the quizmaster and answer judge must ensure that the reference question given is the exact same type as the written question.
+
+The quizmaster should consult with the answer judge(s) at any time they feel is necessary, provided it does not infringe upon the quizzer's question answer duration period.
+
+#### Quizmaster Prompts
+
+During the question answer duration period, the quizmaster will use only the following prompts.
+
+##### Pronoun Clarification
+
+If a quizzer has provided all the information in the answer, except the identification of a pronoun present which is part of the answer text, the quizmaster will ask the quizzer to identify the specific pronoun. For example: "Can you identify 'He'?" or "Can you identify 'They'?"
+
+The quizmaster is not allowed to say anything other than what the quizzer has already said when asking for the identification. A pronoun may need to be identified only if the antecedent is in context.
+
+Once the quizzer has completed all information in the question and answer, the quizmaster will prompt the quizzer to identify the specific pronoun.
+
+##### Reference Questions
+
+On a Reference type question, if the quizzer has given all the information in both the question and the answer and the complete question was not read, the quizmaster will ask, "What is your question?"
+
+##### Answer Correction
+
+On a Quote/Finish type question, once the quizzer has quoted to the end of the required verse or verses, if the quote is not yet word-perfect, the quizmaster will say, "Again." This will occur as often as needed until the quizzer quotes the verse perfectly or the question answer duration period expires. The quizmaster is not allowed to say "again" until the quizzer has come to the end of the verse, each time through, as this would give the quizzer information about where their mistake was made.
```

Comment:
I think this would still be FE, but this occurrence of the word "quotes" is the only place it appears (in reference to a quizzer answering). Since we have issues with QMs using "quoting vs. answering" reasoning when ruling, do we want to change "quotes" to "answers" or "responds?"

Reply:
Agreed.

Reply:
db77867